tasks/src/tabular-classification/data.ts

```diff
@@ -6,12 +6,8 @@ import { TASKS_MODEL_LIBRARIES } from "../const";
 const taskData: TaskData = {
 	datasets: [
 		{
-			description: "Binary classification dataset based on a census on income.",
-			id:          "scikit-learn/adult-census-income",
-		},
-		{
-			description: "Multi-class dataset on iris flower species.",
-			id:          "scikit-learn/iris",
+			description: "A comprehensive curation of datasets covering all benchmarks.",
```

Comment:
Should we bring one one of the above? That way we'll have two datasets

Reply:
the benchmark already has those datasets.